new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -731,6 +737,162 @@ public void testCreateNewCluster() {
 
   }
 
+  /**
+   * Test changes the WebLogic credentials and verifies the servers can startup and function with changed credentials.
+   * a. Creates new WebLogic credentials using WLST.
+   * b. Creates new Kubernetes secret for WebLogic credentials.
+   * c. Patch the Domain Resource with new credentials, restartVerion and introspectVersion.
+   * d. Verifies the servers in the domain restarted and accessing the admin server console with new password works.
+   * e. Verifies the the admin server console access with old credentials fail.
+   */
+  @Order(3)
+  @Test
+  @DisplayName("Test change WebLogic admin credentials for domain running in persistent volume")
+  public void testCredentialChange() {
+
+    final String domainUid = "mydomain";
+
+    final String adminServerName = "admin-server";
+    final String adminServerPodName = domainUid + "-" + adminServerName;
+
+    final String managedServerNameBase = "ms-";
+    String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+    int replicaCount = 3;
+
+    logger.info("Getting node port for default channel");
+    int adminServerT3Port = getServiceNodePort(introDomainNamespace, adminServerPodName + "-external", "t3channel");
+
+    // get the pod creation time stamps
+    LinkedHashMap<String, DateTime> pods = new LinkedHashMap<>();
+    // get the creation time of the admin server pod before patching
+    DateTime adminPodCreationTime = getPodCreationTime(introDomainNamespace, adminServerPodName);
+    pods.put(adminServerPodName, adminPodCreationTime);
+    // get the creation time of the managed server pods before patching
+    for (int i = 1; i <= replicaCount; i++) {
+      pods.put(managedServerPodNamePrefix + i,
+          getPodCreationTime(introDomainNamespace, managedServerPodNamePrefix + i));
+    }
+
+    // create a temporary WebLogic WLST property file
+    File wlstPropertiesFile = assertDoesNotThrow(() -> File.createTempFile("wlst", "properties"),
+        "Creating WLST properties file failed");
+    Properties p = new Properties();
+    p.setProperty("admin_host", K8S_NODEPORT_HOST);
+    p.setProperty("admin_port", Integer.toString(adminServerT3Port));
+    p.setProperty("admin_username", ADMIN_USERNAME_DEFAULT);
+    p.setProperty("admin_password", ADMIN_PASSWORD_DEFAULT);
+    p.setProperty("new_admin_user", ADMIN_USERNAME_PATCH);
+    p.setProperty("new_admin_password", ADMIN_PASSWORD_PATCH);
+    p.setProperty("test_name", "replace_admin_user");
+    assertDoesNotThrow(() -> p.store(new FileOutputStream(wlstPropertiesFile), "wlst properties file"),
+        "Failed to write the WLST properties to file");
+
+    // changet the admin server port to a different value to force pod restart
```

Comment:
typo - change, does the comment apply to the use case?

Reply:
fixed